developer_setup.md

```diff
@@ -37,8 +37,17 @@
   sudo passwd postgres
   # ↑ will ask to choose a password
   sudo postgresql-setup initdb
+  # if the above usage of postgresql-setup fails with 'command not found', use the following command with the relevant postgres 9.x version - below an example for postgres 9.1:
```

Comment:
I tried to find any official notice on deprecation, but didn't find
anything. And since I am not checking instructions on all operating
systems, I preferred the "caution mode notice"
On Feb 18, 2016 5:59 PM, "Jason Frey" <notifications@github.com> wrote:

> In developer_setup.md
> <https://github.com/ManageIQ/guides/pull/106#discussion_r53334520>:
>
> > @@ -37,8 +37,17 @@
> >    sudo passwd postgres
> >    # ↑ will ask to choose a password
> >    sudo postgresql-setup initdb
> > +  # if the above usage of postgresql-setup fails with 'command not found', use the following command with the relevant postgres 9.x version - below an example for postgres 9.1:
>
> If it's completely deprecated, then line 39 should just go away.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/guides/pull/106/files#r53334520>.
>